custom_components/zha_map/neighbour.py

```diff
@@ -119,13 +119,17 @@ def _update_info(self):
         """Scan for neighbours."""
         idx = 0
         while True:
-            status, val = await self.device.zdo.request(
-                zdo_t.ZDOCmd.Mgmt_Lqi_req, idx, tries=3, delay=1
-            )
-            self.debug("neighbor request Status: %s. Response: %r", status, val)
-            if zdo_t.Status.SUCCESS != status:
-                self.supported = False
-                self.debug("device does not support 'Mgmt_Lqi_req'")
+            try:
+                status, val = await self.device.zdo.request(
+                    zdo_t.ZDOCmd.Mgmt_Lqi_req, idx, tries=3, delay=1
+                )
+                self.debug("neighbor request Status: %s. Response: %r", status, val)
+                if zdo_t.Status.SUCCESS != status:
+                    self.supported = False
+                    self.debug("device does not support 'Mgmt_Lqi_req'")
+                    return
+            except asyncio.TimeoutError:
+                self.debug("'Mgmt_Lqi_req' timedout")
```

Comment:
```suggestion
                self.debug("[%s] 'Mgmt_Lqi_req' timedout", self.device.nwk)
```

Reply:
Shouldn't the nwk log be added to the log function instead? It is already printing the IEEE.

Reply:
yep, you are right. I thought we were not logging any id at all. This looks good.